rfc/yyyymmdd-template.md

```diff
@@ -0,0 +1,39 @@
+# RFC Title
```

Comment:
Should we add more sections to the RFC template? The way I see it, this template is a guide to how to write an RFC, and having more specific sections might help the writer think about things he might have missed.
For example, we can add sections like:
Motivation, new/changed workflows (how the user will experience it, and a technical description on how to implement), user stories, if this feature is based on an existing implementation in the codebase, breakdown into tasks, risks, etc.

Reply:
Github double-posted @ollevche's response and I attempted to delete one copy and accidentally removed both.

Reply:
I was saying that the current version covers all the core RFC components, except implementation tracking (issue decomposition and status updates). And I think it's a good idea to have a separate issue to handle that part.

Reply:
@ollevche I agree that most of what I said is mentioned in some way in this template RFC. But it's a little vague and could be more detailed (for example, adding sub-headers with sub-topics can give this template more structure and IMO more guidance to the user).
But this is a long discussion that we don't have to figure out right now. We can keep enhancing our RFC template over time. Just wanted to raise this as something to think about.
So I'll approve and we can keep thinking about this, especially during dog-fooding.

Reply:
I tried to leave it a bit vague as historically RFCs have spanned a wide range of topics / proposals.  As we get more experience I'm 100% on board with expanding our recommendations here!